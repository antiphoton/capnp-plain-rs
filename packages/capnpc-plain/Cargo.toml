[package]
name = "capnpc-plain"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
anyhow = "1.0.80"
capnp-plain = { path = "../capnp-plain" }
convert_case = "0.6.0"
once_cell = "1.19.0"
prettyplease = "0.2.16"
proc-macro2 = "1.0.78"
quote = "1.0.35"
serde = { version = "1.0.196", features = [ "derive" ]}
syn = "2.0.50"
