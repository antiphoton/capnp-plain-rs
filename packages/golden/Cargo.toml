[package]
name = "capnp-plain-golden"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
anyhow = "1.0.82"
capnp-plain = { path = "../capnp-plain" }
serde = { version = "1.0.200", features = [ "derive" ]}
serde_json = "1.0.116"
